elastic/kibana
# [SecuritySolution][Onboarding] Send Telemetry when header/footer cards are clicked

## Summary

#196145

To verify:


1. Add these lines to kibana.dev.yml
```
logging.browser.root.level: debug
telemetry.optIn: true
```
\
2. In the onboarding hub, click on header cards.
It should log `onboarding_card_${cardId}` on cards clicked.
<img width="1101" alt="Screenshot 2024-10-16 at 10 30 58" src="https://github.com/user-attachments/assets/902848f2-fdc5-412d-bfe0-9ed51ba87c56">
<img width="1258" alt="Screenshot 2024-10-15 at 16 54 32" src="https://github.com/user-attachments/assets/883a49a2-cd78-4438-91bb-21b2842b8893">

\
3. It should log `onboarding_footer_link_${footerLinkId}` on footer links visited.
<img width="1019" alt="Screenshot 2024-10-16 at 10 31 26" src="https://github.com/user-attachments/assets/a7ff80a7-a30d-42e9-84d3-5a14fd243022">

<img width="1200" alt="Screenshot 2024-10-15 at 17 29 59" src="https://github.com/user-attachments/assets/3034ca61-425b-47f5-a415-8bf6065f2c6f">


### Checklist

Delete any items that are not applicable to this PR.
